### PR TITLE
Test stubs

### PR DESCRIPTION
**Pull request by reillysiemens**
### Description

This stubs out most of the tests for assignments three, four, and five. A few tests are left in, but all of them fail until a line or two is removed. There are also some new comments and rabbit holes.

### Check List

- [ ] All test passed
- [ ] Any new functionality or fixes you added is covered by new tests

All the tests are supposed to fail. Still feels weird...

### CODE REVIEW

File: tests/assignment_five/test_arbitrary_operators.py [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+    # When you're done just remove the line below this comment and that's it.
+    assert False
+
+
+def test_calc_raise_error_when_parsing_adjacent_operators():
+    """
+    Test that ``Calc.parse()`` raises a ``CalcError`` if a given expression
+    contains adjacent operators. Test that the error message is correct.
+    """
+    assert False  # You're on your own for this one. See above for an example.
```
Comment (rawrgulmuffins) [inline tests/assignment_five/test_arbitrary_operators.py]: I like it.

File: tests/assignment_four/test_arithmetic.py [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+
+
+def test_calc_parses_integer_division_instead_of_floating_point_division():
+    """
+    Test that ``Calc.parse()`` does integer division rather than floating point
+    division. That is to say, parsing an expression that has a fractional
+    remainder should yield a truncated result.
+
+    Note:
+        Yes, this is an obscenely long function name.
```
Comment (rawrgulmuffins) [inline tests/assignment_four/test_arithmetic.py]: =P

Comment (reillysiemens) [inline tests/assignment_four/test_arithmetic.py]: While I believe identifiers are unlimited in length 68 characters seems... excessive...

**Comment by rawrgulmuffins** [comment]
This looks really clean to me. I like the test renames.

**Pull request merged**